fpp

```diff
@@ -25,6 +25,9 @@ PYTHONCMD="python"
 function doProgram {
   # process input from pipe and store as pickled file
   $PYTHONCMD "$BASEDIR/src/processInput.py" "$@"
+  # if it failed, just fail now and exit the script
+  # this works for the looping -ko case as well
+  if [[ $? != 0 ]]; then exit $?; fi
```

Comment:
Followup from #131 -- @lastquestion do you know why this works for `processInput.py` but not `choose.py`? I'm assuming its because we connect to `dev/tty` or from closing stdin or something